src/routes/+layout.svelte

```diff
@@ -1,22 +1,27 @@
 <script>
   import '../app.postcss'
+  import { page } from '$app/stores'
   import Footer from '$lib/components/footer.svelte'
   import Header from '$lib/components/header.svelte'
   import logoDesktop from '$lib/assets/logo-home.svg'
   import logoMobile from '$lib/assets/diagonal-logo.svg'
   import RingText from '$lib/components/ring-text.svelte'
+
+  const desiredUrl = '/'
+
+  $: hasgradient = $page.url.pathname === desiredUrl
 </script>
 
 <div class="container">
   <Header />
-  <div class="logo-mobile">
+  <div class={hasgradient ? 'logo-desktop' : 'logo-desktop gradient'}>
```

Comment:
this is a bit stupid, could be: `class="logo-desktop {hasGradient ? 'gradient':''}"